pkg/project/project.go

```diff
@@ -5,9 +5,15 @@ var (
 	gitSHA      = "n/a"
 	name        = "app-operator"
 	source      = "https://github.com/giantswarm/app-operator"
-	version     = "n/a"
+	version     = "1.0.1"
```

Comment:
Please note this is a different kind of version from the below `AppVersion`

Reply:
This is the version of the operator. It's different because all app CRs currently have a version of 1.0.0. This is explained in the comment below.

I think we need to revisit this after Helm 3. So we run multiple app-operator instances linked to a release.